src/main/java/org/mule/module/apikit/ApikitExtensionLoadingDelegate.java

```diff
@@ -141,6 +150,26 @@ public void accept(ExtensionDeclarer extensionDeclarer, ExtensionLoadingContext
     addConfigRefParameter(consoleDeclarer, apikitConfigStereotype);
   }
 
+  /**
+   * Sets the configuration factory, which is required and expected to be present by the Java SDK.
+   */
+  private ConfigurationDeclarer withConfigurationFactory(ConfigurationDeclarer apikitConfig,
+                                                         ApiKitConfigFactory apiKitConfigFactory) {
+    try {
+      // Since this class is internal to the container classlaoder, we cannot execute it directly because this code runs with the
```

Comment:
Typo
```suggestion
      // Since this class is internal to the container classloader, we cannot execute it directly because this code runs with the
```